frontend/src/scss/PrimeStyles.scss

```diff
@@ -130,6 +130,24 @@
   background-color: color($theme-color-prime-red-lightest);
 }
 
+.prime-radio--horizontal {
+  list-style-type: none;
+  margin-top: 0;
+  padding: 0;
+  text-align: center;
+}
+.prime-radio--horizontal__container {
+  position: relative;
+  display: list-item;
+  width: 45%;
+  max-width: 5rem;
+  float: left;
+  margin: 0;
+}
+.prime-radio--horizontal__container:first-child {
+  margin-right: units(1);
```

Comment:
Should we be using the USWDS v2 design tokens consistently? If so I'll need to study up on it. I have had a hard time even understanding the logic behind them, much less use them. For example, why wouldn't the width and max-width above this also use design tokens? If they do, what value would they have?

Reply:
I stole this, along with most of our css and commonComponents, from other USDS projects where designers introduced css. I don't actually know how styles in USWDS works, lol